### DISCUSSION TITLE
Welcome to the Discussions of PublicCode.yml!

### DISCUSSION THREAD

**Discussion by ruphy** [publiccodeyml/publiccode.yml#104] [Announcements]
Dear all,

in spite of a complicated year, we're finally ready to kickstart the working group to transition the development of the format to the open. While we've been setting up the official working group, Github has introduced the "discussion" feature, which I believe could be enough for the working of our community. If true, it could be a game changer for us as it will allow us to simplify our infrastructure and make the maintenance of a separate forum redundant.

I would like to propose an introductory call to see each other in video, discuss how to move forward, and effectively mark the official start of the international working group.

I am proposing some slots, hoping that we could find a date which works for all without going too far in time! https://doodle.com/poll/ebdntc55r6ahvrnt

If you are the technical delegate for your Country, please add a 👍  reaction to this message, so that we are sure you have seen it, otherwise we'll e-mail-poke you. :-)

See you soon!

@shimonshore @zorp @bzg
@cknebel @giosce @devinbalkind @bvhme @ericherman @ainali
@fiorellabgs @nclsppr @fnogatz

**Comment by bzg** [comment]
Dear Riccardo, dear all,

thank you very much for adding me to this group, I'm looking forward participating to the discussions.

Here is a very quick update on France's plans:

- deploy a french version of https://publiccode-editor.developers.italia.it
- test it with a few French public code projects
- start referencing projects with a publiccode.yml separately on https://code.gouv.fr
- define the core value Etalab's Free Software unit can provide to these projects
- iterate

We plan to set this up during the first semester of 2022, so the creation of this channel is quite timely.

Cheers!

**Reply by libremente** [reply]
Amazing @bzg! Thanks a lot for your efforts, for any issue please do not hesitate to contact us. Happy hacking 😉

**Reply by ruphy** [reply]
@bzg getting back to you on this - would you like to give everyone a quick heads up on what France intends to do next?

**Reply by bzg** [reply]
Well, it would be premature right now to really talk about a "plan" (with strong commitments), but the list above reflects our current _intentions_.  Sure, I can give a quick heads up about this!

**Comment by ruphy** [comment]
Dear all,

it looks like a date was found!

We are going to have our very first call on the 15th of December at 3PM GMT/4PM CET/... (i.e. in exactly one week and 10 minutes from now): https://www.inyourowntime.zone/2021-12-15_16.00_Europe.Berlin

I'll announce the call in its own thread too and take it from there.

I'm excited to meet all of you!! 😄
